tritium-processor/src/test/resources/com/palantir/tritium/examples/ProxiedBindsParameter.java.generated

```diff
@@ -0,0 +1,74 @@
+package com.palantir.tritium.examples;
+
+import java.lang.reflect.InvocationHandler;
+import java.lang.reflect.Method;
+import java.lang.reflect.UndeclaredThrowableException;
+import javax.annotation.processing.Generated;
+
+@Generated("com.palantir.tritium.processor.ProxyAnnotationProcessor")
+public final class ProxiedBindsParameter implements BindsParameter {
```

Comment:
Bikeshed: how about "BindsParameterProxy"? "BindsParameterProxyFactory" (because you can never have enough factories)?